Makefile

```diff
@@ -44,7 +44,7 @@ $(lib_dir)/openh264/libopenh264.x86_64-darwin.a: $(src_dir)/openh264
 
 .PHONY: cross-libraries
 cross-libraries:
-	docker build -t mediadevices-builder -f cvendor/builder.Dockerfile .
+	docker build -t mediadevices-builder -f builder.Dockerfile .
```

Comment:
Something like `libs-builder.Dockerfile` might be better?

Reply:
sounds good. I'll rename it.